src/patient-registration/patient-registration.component.tsx

```diff
@@ -142,6 +155,34 @@ export const PatientRegistration: React.FC = () => {
             </div>
             <DemographicsSection setFieldValue={props.setFieldValue} values={props.values} />
             <ContactInfoSection />
+
+            <FieldArray name="attributes">
+              {fieldArrayProps => {
+                const { push, form } = fieldArrayProps;
+                const { values } = form;
+                const { attributes } = values;
+
+                return personAttConfig.length ? (
```

Comment:
```suggestion
                return config.personAttributes.length ? (
```

etc. for the other usages

Reply:
Fixed